kin-backup-and-restore/kin-backup-and-restore-ui-tests/src/androidTest/kotlin/kin/backup/and/restore/ui/tests/RestoreActivityFullHappyPathTest.kt

```diff
@@ -0,0 +1,111 @@
+package kin.backup.and.restore.ui.tests
+
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.action.ViewActions.*
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.intent.rule.IntentsTestRule
+import android.support.test.espresso.matcher.ViewMatchers.*
+import android.support.test.filters.LargeTest
+import android.support.test.runner.AndroidJUnit4
+import kin.backup.and.restore.ui.tests.UiTestUtils.childAtPosition
+import kin.backup.and.restore.ui.tests.UiTestUtils.intendingStubQRIntent
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.allOf
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.io.IOException
+
+
+@LargeTest
+@RunWith(AndroidJUnit4::class)
+class RestoreActivityFullHappyPathTest {
+
+    @get:Rule
+    var activityTestRule = IntentsTestRule(BackupAndRestoreDummyActivityForUITest::class.java)
+
+    @Before
+    @Throws(IOException::class)
+    fun setup() {
+        intendingStubQRIntent()
+    }
+
+    @After
+    fun tearDown() {
+    }
+
+    @Test
+    fun restoreActivityFullHappyPathTest() {
+        val appCompatButton = onView(
+                allOf(withId(R.id.restore_account), withText("Restore Account"),
```

Comment:
can't the text rely on resources?

Reply:
Yes it can, probably forgot to do it here